interoperability-layer-openhim/package-metadata.json

```diff
@@ -7,9 +7,17 @@
   "dependencies": [],
   "environmentVariables": {
     "OPENHIM_CORE_IMAGE": "jembi/openhim-core:microfrontends-pre-release",
+    "OPENHIM_CONSOLE_IMAGE": "jembi/openhim-console:v1.18.2",
```

Comment:
Im wondering if this image tag for console shouldnt be pointing the the new microfrontends one?